frontend/src/app/localization.tsx

```diff
@@ -297,5 +297,141 @@ export const elementLocalization: ElementLocalization = {
+                <li>Contacto físico inapropiado o indeseado.</li>
+                <li>Intimidación o bullying, ya sea en línea o en persona.</li>
+                <li>
+                    Cualquier otro tipo de comportamiento considerado inapropiado en un entorno profesional.
+                </li>
+            </ul>
+            <h2>Consecuencias de un comportamiento inaceptable</h2>
+            <p>
+                Lxs organizadores del guadala<b>hacks</b> tienen el derecho y la responsabilidad
+                de sancionar o expulsar temporal o permanentemente a cualquier participante por
+                cualquier comportamiento que consideren inapropiado, amenazante, ofensivo o dañino.
+            </p>
+            <h2>Si eres testigo o eres víctima de un comportamiento inaceptable</h2>
+            <p>
+                Si eres víctima o testigo de un comportamiento inaceptable, o tienes alguna otra inquietud,
+                por favor comunícate con un miembro del equipo de guadala<b>hacks</b> lo antes posible
+                o escríbenos a <a href="mailto:incidentes@guadalahacks.com" style={{color: "#E4007C"}}>
+                    incidentes@guadalahacks.com
+                </a>.
+            </p>
+            <h2>Respeto al entorno</h2>
+            <p>
+                Recordamos a los participantes que deben respetar las instalaciones y su entorno,
+                así como cualquier equipo y recurso proporcionado como parte del hackatón.
+            </p>
+            <h2>Contacto</h2>
+            <p>
+                Para reportar problemas relacionados con el código de conducta, comunícate con
+                un miembro del equipo de guadala<b>hacks</b> o escríbenos a
+                <a href="mailto:incidentes@guadalahacks.com" style={{color: "#E4007C"}}>
+                    incidentes@guadalahacks.com
+                </a>.
+            </p>
+            <p>
+                Apreciamos tu colaboración en hacer de este guadala<b>hacks</b> un lugar amigable y
+                acogedor para todxs lxs participantes.
+            </p>
+        </>,
+        en: <>
+            <h1>The guadala<b>hacks</b> Code of Conduct</h1>
+            <h2>Our commitments</h2>
+            <p>
+                The goal of guadala<b>hacks</b> is to foster innovation and the creation of
+                tech solutions in a space where we can build, contribute and learn.
+                We're committed to provide a safe, welcoming environment for everyone, regardless
+                of gender, sexual orientation, disability, ethnical background, religion, or any
+                other diversity category. This code of conduct outlines the behavior guidelines
+                for all attendands and staff members, as well as the consequences of breaking it.
+            </p>
+            <h2>Expected behavior</h2>
+            <ul>
+                <li>Participate as your authentic self and in an active manner.</li>
```

Comment:
maybe "Participate with authenticity and in an active manner"? Because this refers both to identity and integrity while competing